models/eqV2_tracegrad/test_fairchem_kappa.sh

```
@@ -0,0 +1,21 @@
+export PYTHONPATH=/data/fywang/code/fairchem-tracegrad-hanhai/src:$PYTHONPATH
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)